docs/intro/languages/other.mdx

```diff
@@ -1 +1,40 @@
-TODO
+Nitric is an open framework designed to support additional languages for multi-cloud development. Languages are supported through SDKs which provide Nitric APIs to cloud services and a FaaS micro-framework to build cloud functions.
+
+If you want add support for another language please contribute to the [GitHub Nitric](https://github.com/nitrictech) open source project. Please reach out to us, we would will be super happy to help.
+
+## Building SDKs
+
+Nitric language SDKs are layered with 2 parts:
+
+1. API library which contains generated gRPC code (base Nitric API)
+2. SDK library which provides a FaaS micro-framework and higher level Fluent Nitric API on top of the base Nitric API
+
+[gPRC](https://grpc.io/) is used to provide APIs with native language support and high performance.
+
+Application code uses SDKs Nitric APIs to call cloud services via the Nitric Membrane.
+
+<img
+  height="120"
+  src="../../../assets/img/nitric-sdk-layers.svg"
+  alt="nitric sdk architecture diagram"
+/>
+
+Currently supported Nitric language APIs and SDKs are provided below:
+
+| Node                                                             | Python                                                               | JVM                                                            | Go                                                           |
+| ---------------------------------------------------------------- | -------------------------------------------------------------------- | -------------------------------------------------------------- | ------------------------------------------------------------ |
+| [Node API](https://github.com/nitrictech/apis/tree/develop/node) | [Python API](https://github.com/nitrictech/apis/tree/develop/python) | [JVM API](https://github.com/nitrictech/apis/tree/develop/jvm) | [Go API](https://github.com/nitrictech/apis/tree/develop/go) |
+| [Node SDK](https://github.com/nitrictech/node-sdk)               | [Python SDK](https://github.com/nitrictech/python-sdk)               | [JVM SDK](https://github.com/nitrictech/java-sdk)              | [Go SDK](https://github.com/nitrictech/go-sdk)               |
+
+To add new language support we would recommend you follow the same layering pattern with a generated gRPC base API library and a separate SDK repository which provides a higher level language support developers to use.
```

Comment:
Suggest replacing: "which provides a higher level language support developers to use"
With: "which provides higher-level language support for developer use" (or something similar)